traffic_monitor/tests/Dockerfile-golangtest

```diff
@@ -23,6 +23,6 @@ WORKDIR /go/src/$DIR/traffic_monitor
 
 RUN bash -c "go get -u github.com/jstemmer/go-junit-report"
 RUN bash -c "go get -v"
-CMD bash -c 'go test -v ./... ../lib/go-tc/... 2>&1 | go-junit-report --package-name=golang.test.tm --set-exit-code > /junit/golang.test.tm.xml && chmod 777 -R /junit && cat /junit/golang.test.tm.xml'
-#
```

Comment:
I think you wanna leave this in. I don't use vim regularly enough to know for sure, but I think this empty comment line may be necessary for it to pick up the following syntax line.